src/init.cpp

```diff
@@ -709,6 +709,10 @@ bool InitSanityCheck(void)
 
 bool AppInitServers(boost::thread_group& threadGroup)
 {
+    RegisterAllCoreRPCCommands(tableRPC);
```

Comment:
Nit, could be in `OnRPCStarted`, like there is no need to register if it fails to start?